tests/e2e/tests/upgrade/upgrade_test.go

```diff
@@ -223,30 +232,13 @@ func applyRules(ruleKeys []string) error {
 	return nil
 }
 
-func checkTraffic(t *testing.T) {
```

Comment:
why is this gone. what is the replacement ?

Reply:
It has been merged into probeGateway as there were some duplicate logic there.